docs/style_guide.md

```diff
@@ -0,0 +1,28 @@
+# Style Guide
+
+This project follows Google's C++ style guide
+
+https://google.github.io/styleguide/cppguide.html
```

Comment:
Make this a link in the prior line.

`This project follows [Google's C++ style guide](https://google.github.io/styleguide/cppguide.html).`